static/app/utils/metrics/mri.tsx

```diff
@@ -0,0 +1,93 @@
+import * as Sentry from '@sentry/react';
+
+import {t} from 'sentry/locale';
+import {MetricType, MRI, ParsedMRI} from 'sentry/types/metrics';
+import {parseFunction} from 'sentry/utils/discover/fields';
+
+import {DEFAULT_METRIC_ALERT_AGGREGATE, UseCase} from './index';
+
+export const DEFAULT_MRI: MRI = 'c:custom/sentry_metric@none';
+
+export function isMRI(mri?: unknown): mri is MRI {
+  return !!parseMRI(mri);
+}
+
+export function parseMRI(mri?: unknown): ParsedMRI | null {
+  if (!mri) {
+    return null;
+  }
+  try {
+    return _parseMRI(mri as MRI);
+  } catch (e) {
+    return null;
+  }
+}
+
+function _parseMRI(mri: MRI): ParsedMRI {
+  const mriArray = mri.split(new RegExp(/[:/@]/));
+
+  if (mriArray.length !== 4) {
+    Sentry.captureMessage(`Invalid MRI: ${mri}`);
+    throw new Error('Invalid MRI');
+  }
+
+  const [metricType, useCase, name, unit] = mriArray;
+
+  return {
+    type: metricType as MetricType,
+    name,
+    unit,
+    useCase: useCase as UseCase,
+  };
+}
+
+export function toMRI({type, useCase, name, unit}: ParsedMRI): MRI {
+  return `${type}:${useCase}/${name}@${unit}`;
+}
+
+export function formatMRI(mri: MRI): string {
+  return parseMRI(mri)?.name ?? mri;
+}
+
+export function getUseCaseFromMRI(mri?: string): UseCase {
+  const parsed = parseMRI(mri);
+
+  return parsed?.useCase ?? 'sessions';
```

Comment:
Why is the default `sessions`? 
Shouldn't we just return `null` here and let the default value be the concern of the code that uses it?